UKHomeOffice/cerberus-service
# COP-11346: Add test for Add document number against each co-traveller

## Description
Add test for Add document number against each co-traveller

## To Test
npm run cypress:test:local -- --spec cypress/integration/airpax/airpax-tasks-details.spec.js

Should verify Document number in document column for Co-traveller if present
## Developer Checklist

\* Required
